### ISSUE TITLE
Common suffix

### ISSUE THREAD

**Issue by lukescott** [novemberborn/babel-plugin-import-glob#38]
It looks like this plugin strips a "common prefix" and "common extension". This works well if your file naming conventions use `.` for a common suffix, such as in your example `.handlebars.js`. However, if you prefer to use `-` or `_` instead, you end up with the common bit in the identifier.

It looks like this project uses two custom dependencies:

```
"common-extname": "^1.0.0",
"common-path-prefix": "^1.0.0",
```

So changing this becomes a bit tricky. Do I need to add a third `common-path-suffix` repo?

I think a common suffix would probably be more helpful, as the common suffix will likely be in the pattern anyway. Of course there could be a configuration option to toggle the behavior.

**Comment by novemberborn** [comment]
Interesting. Thinking back I used (and wrote) `common-extname` because I figured removing the extname was the most predictable course of action. What kind of suffixes are you using? Is it about the pattern or are they values that can be hardcoded in the plugin configuration?

**Comment by lukescott** [comment]
You know what, it's probably fine the way it is. I have a plugin structure that looks like this:
```
textField/textField-reducer.js
multipleChoice/multipleChoice-reducer.js
```

If I use `.reducer.js` it emits that portion. What I'm trying to do is create an object that had keys like `textField` and `multipleChoice`. I started with just `textField` and I got that with `textField.reducer.js`, until I added a second one, which now I'm getitng `textField$textField`.

I think no matter what, I'm going to have to pre-process the object. There are too many different ways it could be interpreted, and how it works now is fine.

I really can't find a way to tell it what to use. I wish I could do something like this:

```
import * as reducers from "./(*)/*.reducer.js";
```

Which would extract the wrapped portion. But as far as I know glob doesn't support that.
